server/apps/prepopulate/app_initialize.py

```diff
@@ -50,19 +54,20 @@ class AppInitializeWithDataCommand(superdesk.Command):
     """
 
     option_list = [
-        superdesk.Option('--entity-name', '-n', dest='entity_name', default='')
+        superdesk.Option('--entity-name', '-n', dest='entity_name', default=''),
+        superdesk.Option('--profile', '-p', dest='profile')
```

Comment:
imo better way would be to make directory a param so that we don't have to push this into repo but it can be in some private repo/space and still it could be imported. or to be able to read each entity from a stdin so that you can import it into docker without adding it into an image (but not sure it would be possible like `cat users.json > docker exec -ti manage.py app:initialize_data --entity-name users`)